README.md

```diff
@@ -0,0 +1,33 @@
+# Core Snap
+
+This repository contains the official Core snap. The core snap plays
+
+## Reporting Issues
```

Comment:
s/Issues/issues/ ("Title case", so we're not encouraging awkward capitalization)